platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -840,6 +841,7 @@ public Stream<VirtualLeafRecord<K, V>> deletedLeaves() {
                 final K key = element.key;
                 final Mutation<K, VirtualLeafRecord<K, V>> mutation = lookup(keyToDirtyLeafIndex.get(key));
                 if (mutation != null && mutation.isDeleted()) {
+                    element.value.setPath(Path.INVALID_PATH);
```

Comment:
This is not needed with the new approach

Reply:
Removed.